src/controllers/enrolment/helpers/waitlistedEnrolment.js

```diff
@@ -17,6 +18,25 @@ const waitlistedEnrolment = async (req, session) => {
     { session }
   );
 
+  let {clubMembershipId} = req;
+  let consentFilter = { clubMembershipId };
+  let consentUpdate = {
+    businessId,
+    memberId,
+    clubMembershipId,
+    consent,
+    newsletter,
+  };
+  let consentOption = {
+    new: true,
+    upsert: true,
+  };
+  await MemberConsent.findOneAndUpdate(
```

Comment:
if consent and newsletter are not sent do not update MemberConsent. also you might need to use $set